app/controllers/newsfeed_controller.rb

```diff
@@ -0,0 +1,8 @@
+class NewsfeedController < ApplicationController
+  def show
```

Comment:
An alternative is index, since you're not creating an actual newsfeed, it might be more cruddy to have it as index instead of show.